packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -198,6 +201,10 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       body.device.countryCode = this.webex.meetings.geoHintInfo.countryCode;
       // @ts-ignore
       body.device.regionCode = this.webex.meetings.geoHintInfo.regionCode;
+      body.device.localIp =
```

Comment:
question: I can see this being sent to locus, but where does this appear after this for us to use?

Reply:
Sorry not sure what you mean - why would we need this to appear after this for us to use? we need only send it to locus so that they can capture fraudulent activities.

Reply:
Ah I see, it's for locus use - I thought it might have been for CA / metrics - thanks for clearing that up.